docs/enhancements/20190815-zone-aware-routing.md

```diff
@@ -32,24 +32,24 @@ Teach ingress-nginx about availability zones where endpoints are running in. Thi
 
 ## Motivation
 
-When users run their services across multiple availability zones they usually pay for egress traffic between zones. Providers such as GCP, Amazon EC charges money for that.
-ingress-nginx when picking an endpoint to route request to does not consider whether the endpoint is in different zone or the same one. That means it's at least equally likely
-that it will pick an endpoint from another zone and proxy the request to it. In this situation response from the endpoint to ingress-nginx pod is considered as
-inter zone traffic and costs money.
+When users run their services across multiple availability zones they usually pay for egress traffic between zones. Providers such as GCP, Amazon EC2 charge money for that.
+ingress-nginx when picking an endpoint to route request to does not consider whether the endpoint is in a different zone or the same one. That means it's at least equally likely
+that it will pick an endpoint from another zone and proxy the request to it. In this situation response from the endpoint to the ingress-nginx pod is considered as
```

Comment:
```suggestion
that it will pick an endpoint from another zone and proxy the request to it. In this situation response from the endpoint to the ingress-nginx pod is considered
```